openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -57,6 +57,37 @@ <h2 class="hd hd-3 page-title">${course.display_name_with_default}</h2>
     <div class="page-content">
         <div class="layout layout-1t2t">
             <main class="layout-col layout-col-b">
+
+                % if upgrade_url and upgrade_price:
+                <div class="section">
+                    <div class="vc-message tex2jax_ignore" role="group" aria-labelledby="vc-title" tabindex="-1" style="display: none;">
```

Comment:
Don't use inline styles as that makes it harder for the CSS to be in control.

Reply:
In this case, we don't want CSS to be in control. I want JS to be in control, so we have minimize any weirdness if the user has already collapsed the hero.